src/sql/workbench/contrib/jobManagement/browser/jobHistory.component.ts

```diff
@@ -224,25 +222,17 @@ export class JobHistoryComponent extends JobManagementView implements OnInit {
 				self._stepRows[0].stepId = nls.localize('stepRow.stepID', "Step ID");
 				self._stepRows[0].stepName = nls.localize('stepRow.stepName', "Step Name");
 				self._stepRows[0].message = nls.localize('stepRow.message', "Message");
-				this._showSteps = self._stepRows.length > 1;
+				self._showSteps = self._stepRows.length > 1;
 			} else {
 				self._showSteps = false;
 			}
 			if (self._agentViewComponent.showHistory) {
+				this._jobManagementService.onStepsChange(self._stepRows);
```

Comment:
>this [](start = 4, length = 4)

Don't mix this and self. (self shouldn't be necessary here, but it's fine to leave it since that'd be a lot of extra changes)